check_metadata.py

```diff
@@ -225,9 +244,59 @@ def keywordSDKCheck(words):
 doNotScan = {'AssemblyInfo.cs', 'CMakeLists.txt', 'check_metadata.py', 'movie_data.json'}
 root = './'
 
-# Blacklist of words that should not in include in code samples
+# list of words that should not in include in code samples
 
-blacklist = {'alpha-docs-aws.amazon.com', 'integ-docs-aws.amazon.com'}
+denyList = {'alpha-docs-aws.amazon.com', 'integ-docs-aws.amazon.com'}
+allowList = {
```

Comment:
Were these allowlist strings copied from the current whitelist? There are lots of duplicates in it.

Reply:
These are from paste whitelist requests and things the script flagged during my tests.